editoast/openapi.yaml

```diff
@@ -420,7 +420,7 @@ paths:
                     nullable: true
                     example: null
                     description: The index of the previous page (null if requesting the first page)
-                  results:
+                  result:
```

Comment:
I checked on `dev` and the field name is `results`.

Reply:
Indeed, ... I need to rebuild the docker stack :)
let me fix it